lib/rspec/core/reporter.rb

```diff
@@ -73,7 +73,10 @@ def start(expected_example_count, time=RSpec::Core::Time.now)
       notify :seed, Notifications::SeedNotification.new(@configuration.seed, seed_used?)
     end
 
-    # @private
+    # @api
+    # @param message to send to formatters
```

Comment:
The type spec is optional but I think it's good to be explicit.  If this can be any type of object then `[Object]` would be helpful to add.  Otherwise doc readers won't know if it can be anything or if we forgot to include it in the docs or whatever.

That said, the docs for `MessageNotification` say that it's a string, so we should update it there, too.

Also, we do depend upon the message object having a `to_s` method, because we pass it to `puts`, which calls `to_s` on the provided argument:

https://github.com/rspec/rspec-core/blob/703e74209045845e8ea84b4c45e34bf562a1bfd6/lib/rspec/core/formatters/base_text_formatter.rb#L22-L24

...so I think the type specification we actually want is `[#to_s]`.